website/src/components/Edit/EditPage.spec.tsx

```diff
@@ -116,11 +115,6 @@ describe('EditPage', () => {
 });
 
 const expectTextInSequenceData = {
-    original: (metadata: Record<string, string>): void =>
-        Object.entries(metadata).forEach(([key, value]) => {
-            expect(screen.getByText(key + ':')).toBeInTheDocument();
-            expect(screen.getByDisplayValue(value)).toBeInTheDocument();
-        }),
     originalMetadata: (metadata: UnprocessedMetadataRecord): void =>
```

Comment:
hmm do we still expect this? (I know nothing has changed about it in this PR, but just wondering from before)